README.md

```diff
@@ -113,3 +117,64 @@ The finished image can be found at
 ## Install image
 
 Please refer to our [BISDN Linux docs](https://docs.bisdn.de/getting_started/install_bisdn_linux.html) on how to install the resulting image.
+
+## Known Issues
```

Comment:
Since we can now drop the requirment of having python2 and python2-yaml installed, we will want to add more to this sections in the future.

```suggestion
## Known Issues when building older versions of BISDN Linux
```